meson.build

```diff
@@ -19,6 +19,9 @@ project(
   ],
 )
 
+# NOTE: There are tools in hw/top_englishbreakfast that mess with this line.
```

Comment:
Would you mind to change the language here a bit? It's perfectly sufficient to say that these tools "need" this line.